packages/components/src/local/molecules/chat-channel-message/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import ChatMessageBubble from '../chat-message-bubble'
+/* Import Types */
+import Props from './types/props'
+
+/* Import Stylesheet */
+import styles from './styles.module.scss'
+import { boolean, radios } from '@storybook/addon-knobs'
+import { forceColors } from '@serge/helpers'
+import { forces } from '@serge/mocks'
+import WriteMessagePanel from '../write-message-panel'
+
+
+const forceColorList = forceColors(forces)
+
+const forceValues = {
+  Blue: 'Blue',
+  White: 'White',
+  Red: 'Red'
+}
+
+/* Render component */
+export const ChatChannelMessage: React.FC<Props> = () => {
+  return (
+    <div
+      className={`${styles['chat-channel-message']}`}
+    >
+      <ChatMessageBubble
```

Comment:
data is being hard-coded into this component. It has to come in through props.

Reply:
okay I'll make changes accordingly